server/task_specs/feature-apply.py

```diff
@@ -0,0 +1,147 @@
+
+import csv
+import json
+
+import numpy as np
+from six import PY2
+
+SNIFFER_SIZE = 1024
```

Comment:
Yeah, I was thinking about factoring them out into their own file.  I ended up shelving the idea (for now), but very well may come back to this at a later time.